src/components/pages/Publish/index.tsx

```diff
@@ -24,20 +37,29 @@ import Button from '../../atoms/Button'
 const formName = 'ocean-publish-form'
 
 export default function PublishPage({
-  content
+  content,
+  contentAlgoPublish
 }: {
   content: { warning: string; form: FormContent }
+  contentAlgoPublish: { warning: string; form: FormContent }
 }): ReactElement {
   const { debug } = useUserPreferences()
   const { publish, publishError, isLoading, publishStepText } = usePublish()
   const { isInPurgatory, purgatoryData } = useOcean()
   const [success, setSuccess] = useState<string>()
   const [error, setError] = useState<string>()
+  const [title, setTitle] = useState<string>()
   const [did, setDid] = useState<string>()
   const [publishType, setPublishType] = useState<string>()
 
   const hasFeedback = isLoading || error || success
 
+  useEffect(() => {
+    publishType === 'data'
```

Comment:
just small thing, we should keep value names consistent. So in this case this refers to the `type` set in the DDO and for those it is either `dataset` or `algorithm` so we can use that naming here too